framework-plugins/lisk-framework-http-api-plugin/swagger.yml

```diff
@@ -66,3 +47,768 @@ info:
+          description: Data not found.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        429:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /blocks:
+    get:
+      tags:
+        - Blocks
+      summary: Requests block data
+      description: Search for block in the system by height.
+      produces:
+        - application/json
+      parameters:
+        - name: height
+          in: query
+          description: height of the searching block.
+          required: true
+          schema:
+            type: integer
+      responses:
+        200:
+          description: Block requested.
+          schema:
+            $ref: '#/definitions/BlockResponse'
+        400:
+          description: Malformed query or parameters.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        404:
+          description: Data not found.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        429:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /transactions/{id}:
+    get:
+      tags:
+        - Transactions
+      summary: Requests transaction data
+      description: Search for transaction in the system by ID.
+      produces:
+        - application/json
+      parameters:
+        - name: id
+          in: path
+          description: id of the searching transaction in base64 format
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Transaction requested.
+          schema:
+            $ref: '#/definitions/TransactionResponse'
+        400:
+          description: Malformed query or parameters.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        404:
+          description: Data not found
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        429:
+          description: Too many requests, exceeded rate limit.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        500:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /transactions:
+    post:
+      tags:
+        - Transactions
+      summary: Submits signed transaction for processing.
+      description: |
+        Submits signed transaction object for processing by the transaction pool.
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: body
+          name: transaction
+          description: Transaction object to submit to the network.
```

Comment:
maybe reference to same thing, `Transaction object for processing by the transaction pool`?